sns/parity_math.c

```diff
@@ -101,9 +274,42 @@ static void submatrix_construct(struct m0_matrix *in_mat,
 static void dependency_bitmap_prepare(struct m0_sns_ir_block *f_block,
 				      struct m0_sns_ir *ir);
 
+#if ISAL_ENCODE_ENABLED
+/**
+ * Converts given number of m0_bufvec to m0_buf.
+ * @param[out] bufs  - Array of m0_buf.
+ * @param[in]  bvecs - Array of pointers to m0_bufvec.
+ * @param[in]  count - Number of buffers to copy
+ */
+static int bufvec_to_buf_copy(struct m0_buf *bufs,
+			      struct m0_bufvec **bvecs,
+			      uint32_t count);
+
+/**
+ * Converts given number of m0_buf to m0_bufvec.
+ * @param[out] bvecs - Array of pointers to m0_bufvec.
+ * @param[in]  bufs  - Array of m0_buf.
+ * @param[in]  count - Number of buffers to copy
+ */
+static int buf_to_bufvec_copy(struct m0_bufvec **bvecs,
+			      const struct m0_buf *bufs,
+			      uint32_t count);
```

Comment:
This is a generic function, move it to li/buf.[ch].

Reply:
Added to lib/vec.[ch]